sympy/physics/continuum_mechanics/structure2d.py

```diff
@@ -0,0 +1,1180 @@
+"""
+This module can be used to solve 2D problems with
+singularity functions in mechanics..
+"""
+
+from sympy.core import Basic, Symbol
+from sympy.core.numbers import pi
+from sympy.external import import_module
+from sympy.functions.elementary.complexes import Abs
+from sympy.functions.elementary.miscellaneous import sqrt
+from sympy.functions.elementary.trigonometric import atan2, cos, sin, tan
+from sympy.geometry.polygon import deg, rad
+from sympy.physics.continuum_mechanics.beam import Beam
+from sympy.simplify import nsimplify, simplify
+
+plt = import_module(
+    "matplotlib.pyplot",
+    import_kwargs={
+        "fromlist": [
+            "pyplot",
+        ]
+    },
+)
+
+patches = import_module(
+    "matplotlib.patches",
+    import_kwargs={
+        "fromlist": [
+            "FancyArrow",
+            "Circle",
+            "Rectangle",
+            "Polygon",
+        ]
+    },
+)
+
+np = import_module(
+    "numpy",
+    import_kwargs={
+        "fromlist": [
+            "numpy",
+        ]
+    },
+)
+
+
+class Member:
+    def __init__(self, x1, y1, x2, y2, E, I, A, member_id):
+        self.x1 = simplify(x1)
+        self.y1 = simplify(y1)
+        self.x2 = simplify(x2)
+        self.y2 = simplify(y2)
+        self.E = simplify(E)
+        self.I = simplify(I)
+        self.A = simplify(A)
```

Comment:
`simplify()` should really never be used internally like this, it is more a catchall user function.

Do you mean `simpify()` here?